src/MTGCore.ClientApp/ClientApp/src/components/SearchFilterModal.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <transition name="modal" v-if="ShowModal">
+    <div
+      class="modal"
+      id="TitleModal"
+      style="display: block; background-color: rgba(0, 0, 0, 0.5); transition: opacity 0.3s ease"
+    >
+      <div class="modal-dialog">
+        <div class="modal-content">
+          <div class="modal-header">
+            <h4 class="modal-title">Search filtering options</h4>
+            <button @click="close" type="button" id="btnCloseTitleModal" class="close pull-right">&times;</button>
+          </div>
+
+          <div class="modal-body">
+            <div class="row form-group">
+              <div class="col-lg-2">Type:</div>
+              <div class="col-lg-10">
+                <input v-model="form.Type" type="text" class="form-control" />
+              </div>
+            </div>
+            <hr />
+            <div class="row form-group">
+              <div class="col-lg-2">Colours:</div>
+              <div class="col-lg-10">
+                <select
+                  v-model="form.ColoursOption"
+                  class="form-control form-select form-select-lg"
+                  aria-label="Are exactly"
+                >
+                  <option value="Exactly">Are exactly</option>
```

Comment:
What's the plan for this? Like:
- Exactly
- Including
- Not including
- etc?